db/schema.rb

```diff
@@ -25,8 +36,13 @@
     t.datetime "remember_created_at"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "provider", default: "email", null: false
```

Comment:
Be careful with this... The migration of the previous PR is affecting this one. To avoid this, reset the db with `rails db:drop db:create db:migrate`